lib/log4js.js

```diff
@@ -140,7 +158,8 @@ const log4js = {
   shutdown,
   connectLogger,
   levels,
-  addLayout: layouts.addLayout
+  addLayout: layouts.addLayout,
+  useCallStack,
```

Comment:
I don't think you need the global useCallStack - keeping it at the category level seems cleaner and simpler.

Reply:
fixed